components/automate-ui/src/app/page-components/event-feed-guitar-strings/event-feed-guitar-strings.component.ts

```diff
@@ -389,22 +389,24 @@ export class EventFeedGuitarStringsComponent implements OnInit, OnChanges {
 
     svg.select('.zoom-slider-button.start')
       .call(d3.drag()
-        .on('start', () => { this.dragStarted(); })
-        .on('drag', () => { this.dragStartSlider(); })
-        .on('end', () => { this.dragEnded('start'); }));
+        .on('start', () => this.dragStarted())
+        .on('drag', (e) => this.dragStartSlider(e))
+        .on('end', () => this.dragEnded('start'))
+        );
     svg.select('.zoom-slider-button.end')
       .call(d3.drag()
-        .on('start', () => { this.dragStarted(); })
-        .on('drag', () => { this.dragEndSlider(); })
-        .on('end', () => { this.dragEnded('end'); }));
+        .on('start', () => this.dragStarted())
+        .on('drag', (e) => this.dragEndSlider(e))
+        .on('end', () => this.dragEnded('end'))
+        );
   }
 
   dragStarted() {
     this.sliderWidth = this.getComponentWidth();
   }
 
-  dragStartSlider() {
-    const positionPercentage = d3.event.x / this.sliderWidth * 100;
+  dragStartSlider(e: DragEvent) {
```

Comment:
When I really started to think about it, I feel it makes more sense to use Native event signatures now, since the with update to Version 6, d3 is actually sending native events now as opposed to previously sending d3 specific events.